apps/server/src/modules/tool/context-external-tool/service/context-external-tool-validation.service.ts

```diff
@@ -49,9 +50,7 @@ export class ContextExternalToolValidationService {
 		);
 
 		if (duplicate.length > 0) {
-			throw new ValidationError(
-				`tool_with_name_exists: A tool with the same name is already assigned to this course. Tool names must be unique within a course.`
-			);
+			throw new ContextExternalToolNameAlreadyExistsLoggableException(duplicate[0].displayName);
```

Comment:
If I think about our loggables in this validation class... 
If someone tells use they can not start a tool because of some failure or something else, we would look into the logs.. in my opinion it could also be very helpful if we not only have the displayname or parametername and so on, in addition a unique id which identifies the external/schoolexternal or contextexternal tool. 
This would be really helpful if we want to have some more information from the db instead of "please give the json of the externaltool with the parameter "blub".
What do you think about that?